frontend/packages/core/src/AppLayout/drawer.tsx

```diff
@@ -5,9 +5,9 @@ import {
   Avatar as MuiAvatar,
   Drawer as MuiDrawer,
   List,
-  ListItem,
+  ListItemButton,
   Typography,
-} from "@material-ui/core";
+} from "@mui/material";
```

Comment:
should we update the package.json in the PR as well? when running `make frontend-lint` I'm seeing a few of:

`@mui/material' should be listed in the project's dependencies. Run 'npm i -S @mui/material' to add it  import/no-extraneous-dependencies`

& 

`@mui/icons-material' should be listed in the project's dependencies. Run 'npm i -S @mui/icons-material' to add it  import/no-extraneous-dependencies`

Reply:
They should all be updated, might just be the branch. I've removed all of @material-ui in the `matv5` branch.

Reply:
sg! yeah this was before i realized that the big branch had all the import updates